libs/ui/src/lib/build-status/build-status.component.html

```diff
@@ -66,8 +66,8 @@
         <h3 matSubheader>
           Errors
         </h3>
-        <mat-list-item class="build-error" *ngFor="let error of status?.errors; trackBy: trackByError">
-          {{ error }}
+        <mat-list-item>
+          <pre>{{ status?.errors.join('\n') }}</pre>
```

Comment:
Make sure to use `white-space: pre-wrap;` on this and other pre-tags you are rendering. Also make sure to set the font to `roboto-mono` (you could even do that in our global scss file